src/main/java/org/jboss/aerogear/unifiedpush/utils/PushConfiguration.java

```diff
@@ -65,6 +65,7 @@ public static PushConfiguration read(String location) throws IOException {
                     new InputStreamReader(Thread.currentThread().getContextClassLoader().getResourceAsStream(location)));
             Gson gson = new Gson();
             pushConfiguration =  gson.fromJson(bufferedReader, PushConfiguration.class);
+            pushConfiguration.setServerUrl(!pushConfiguration.getServerUrl().endsWith("/") ? pushConfiguration.getServerUrl() + '/' : pushConfiguration.getServerUrl());
```

Comment:
since w/out the '/' it is already set, no need to reset, no?

I'd just do:
```
            if (!pushConfiguration.getServerUrl().endsWith("/")) {
                pushConfiguration.setServerUrl(pushConfiguration.getServerUrl() + '/');
            }
```